lib/point_quest_web/live/quest.ex

```diff
@@ -4,9 +4,21 @@ defmodule PointQuestWeb.QuestLive do
   """
   use PointQuestWeb, :live_view
 
+  alias PointQuest.Quests.Commands.AddAdventurer
+
   def render(%{live_action: :join} = assigns) do
     ~H"""
-    <div>You're looking to join <%= @session_id %></div>
+    <.form
+      for={@form}
+      id="join-quest-form"
+      phx-change="validate_adventurer"
+      phx-debounce="250"
+      phx-submit="join_party"
+    >
+      <.input type="text" field={@form[:name]} label="Adventurer Name" />
```

Comment:
**Comment (non-blocking)-** can just pass the field name

With changeset backed forms I thought you could just pass `:name` for the field and it works. Is there a reason for the full `@form` you have here.

Reply:
Hrmm... I'll be honest, I didn't try with just the field name, the guide I was following to set up this pattern referenced them fully like so.

I'll attempt the simpler form and report back!

Reply:
Wonder if there is something with the form helpers that I'm supposed to do to enable this? If I try replacing `@form[:name]` with `:name` I get:

> Elixir: attribute "field" in component PointQuestWeb.CoreComponents.input/1 must be a Phoenix.HTML.FormField struct, got: :name